src/main/java/uk/gov/pay/ledger/payout/model/PayoutEntityFactory.java

```diff
@@ -1,18 +1,24 @@
 package uk.gov.pay.ledger.payout.model;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import uk.gov.pay.ledger.event.model.EventDigest;
 import uk.gov.pay.ledger.payout.entity.PayoutEntity;
 import uk.gov.pay.ledger.payout.state.PayoutState;
 
+import java.util.Map;
+
 public class PayoutEntityFactory {
 
     private final ObjectMapper objectMapper;
+    private static final Logger LOGGER = LoggerFactory.getLogger(PayoutEntityFactory.class);
 
-    public PayoutEntityFactory(){
-        this.objectMapper = new ObjectMapper();
-        this.objectMapper.registerModule(new JavaTimeModule());
```

Comment:
I think this is needed otherwise the payout can't serialise `ZonedDateTime` 🤔

Reply:
Tests are fine 🤷‍♂️